test/e2e_node/cpu_manager_test.go

```diff
@@ -611,30 +594,6 @@ func runCPUManagerTests(f *framework.Framework) {
 		err = e2epod.NewPodClient(f).MatchContainerOutput(pod.Name, pod.Spec.Containers[0].Name, expAllowedCPUsListRegex)
 		framework.ExpectNoError(err, "expected log not found in container [%s] of pod [%s]",
 			pod.Spec.Containers[0].Name, pod.Name)
-
-		ginkgo.By("disable cpu manager in kubelet")
-		newCfg = configureCPUManagerInKubelet(oldCfg, &cpuManagerKubeletArguments{
-			policyName:         string(cpumanager.PolicyStatic),
-			enableCPUManager:   false,
-			reservedSystemCPUs: cpuset.CPUSet{},
-		})
-		updateKubeletConfig(f, newCfg, false)
```

Comment:
Does it make sense to keep this test, but instead flip the policy to `none` instead of disabling the feature gate?

Reply:
Possibly yes, let me check.

Reply:
after some experimentation, the test would need some rethinking. We would need to cover the flow on which we change cpumanager policy across restarts, which to date has still some rough edges.
just changing the policy leads to restart failures, very likely _related_ to
```
E1102 06:43:04.969661  125395 cpu_manager.go:223] "Could not initialize checkpoint manager, please drain node and remove policy state file" err="could not restore state from checkpoint: configured policy \"none\" differs from state checkpoint policy \"static\", please drain this node and delete the CPU manager checkpoint file \"/var/lib/kubelet/cpu_manager_state\" before restarting Kubelet"
```
OTOH deleting the state file *in this test* goes against the spirit of the test itself.
All in all, I think downsizing this test and adding more tests (and possibly fixes) in separate PRs serves better the purpose.

Reply:
ACK